jekyll/_cci2/configuration-reference.md

```diff
@@ -2142,7 +2779,3 @@ workflows:
 ```
 {% endraw %}
 
-## See also
```

Comment:
Removed because it doesn't really make sense in the context of this huge reference doc (no one will be reading top to bottom)